RUFAS/time.py

```diff
@@ -95,9 +95,9 @@ def record_time(self) -> None:
             "function": self.record_time.__name__,
             "prefix": "Time",
         }
-        om.add_variable("day", self.day, info_map)
-        om.add_variable("year", self.year, info_map)
-        om.add_variable("calendar_year", self.calendar_year, info_map)
+        om.add_variable("day", self.day, dict(info_map, **{"units": "simulation day"}))
```

Comment:
```suggestion
        om.add_variable("day", self.day, dict(info_map, **{"units": "ordinal day"}))
```
I think this is a little more correct. "Simulation day" does not have a precise meaning, but we could strictly define it for RuFaS if we feel it is appropriate.